File: 0274-h-index/0274-h-index.py
class Solution(object):
    def hIndex(self, c):
        n = len(c)
        c.sort()
        ans = 0
        for i in range(0 , n):
            p = n - i
            if c[i] >= p:
                ans = max(ans , p)
        return ans
        

File: 0274-h-index/0274-h-index.cpp
class Solution {
public:
    int hIndex(vector<int>& nums) {
        int n = nums.size();
        sort(nums.begin() , nums.end());
        int ans = 0;
        for(int i = 0 ; i < n ;i++){
            int paper = n - i;
            if(nums[i] >= paper){
                ans = max(paper , ans);
            }
        }
        return ans;
    }
};